mantidproject/mantid
# Bug in MultidomainFunction python wrapper

Generic function that creates fit function wrappers for composite functions uses correct arguments now.

**To test:**

* Use the script form the issue description. Print out the created fit function to see that the global parameters are tied to each other.
* Run the code from @AnthonyLim23's comment. Gradient 1 and Gradient 2 must be both equal to 3.5

Fixes #21102

**Release Notes** 

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
